browsers/opera_android.json

```
@@ -357,6 +357,7 @@
         },
         "65": {
           "release_date": "2021-09-30",
```

The blog post is dated Oct 20, 2021, so should we change this then?

```suggestion
          "release_date": "2021-10-20",
```